### DISCUSSION TITLE
Metamask wallet is not showing on mobile

### DISCUSSION THREAD

**Discussion by kryptoDev0777** [community/community#48593] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello everyone.
I am using Blocknative onboard to connect my wallet.
I created injected from injectedModule() and added it on wallet lists.
it works very well on desktop, but metamask is not showing on mobile.

(if I disable metamask chrome extension on desktop, metamask is not showing onboard. I am wondering this is the reason.
but it is not showing on mobile even I install metamask app)
Thanks in advance

**Comment by Liberalite** [comment]
Experiencing the same issue. 
Metamask works in desktop but it does not show up on my android mobile phone. 
The only way I can actually connect it is with WalletConnect!
Hoping there is another way for it to appear directly in the browser if that is even possible anymore!
